src/utils.jl

```diff
@@ -177,25 +205,24 @@ num_vertices(p::MakiePlotter) = size(p.physical_coords,1)
 TODO this looks faulty...think harder.
 """
 # Helper to count triangles e.g. for preallocations.
-ntriangles(cell::Ferrite.AbstractCell{2,3,3}) = 1 # Tris in 2D
-ntriangles(cell::Ferrite.AbstractCell{3,3,1}) = 1 # Tris in 3D
-ntriangles(cell::Ferrite.AbstractCell{dim,N,4}) where {dim,N} = 4 # Quads in 2D and 3D
-ntriangles(cell::Ferrite.AbstractCell{3,N,1}) where N = 4 # Tets as a special case of a Quad, obviously :)
-ntriangles(cell::Ferrite.AbstractCell{3,N,6}) where N = 6*4 # Hex
+ntriangles(cell::TriangleBasedCells)      = 1 # Tris in 2D and 3D
+ntriangles(cell::QuadrilateralBasedCells) = 4 # Quads in 2D and 3D
+# ntriangles(cell::Ferrite.AbstractCell{3,N,1}) where N = 4 # Tets as a special case of a Quad, obviously :)
```

Comment:
Can be removed now that embedded cells do not have the case distinction anymore.

```suggestion
```